extensions/wikia/FlowTracking/scripts/createNewPageFlows.js

```diff
@@ -6,23 +6,31 @@ define(
 			articleId = mw.config.get('wgArticleId');
 
 		function trackOnEditPageLoad(editor) {
+			var qs = new QueryString(),
+				// 'flow' is the parameter passed in the url if user has started a flow already
+				flowParam = qs.getVal('flow', false);
+
 			// Track only creating articles (wgArticleId=0) from namespace 0 (Main)
 			// IMPORTANT: on Special:CreatePage even after providing article title the namespace is set to -1 (Special Page)
 			if (namespaceId === 0 && articleId === 0) {
-				var qs = new QueryString(),
-					// 'flow' is the parameter passed in the url if user has started a flow already
-					flowParam = qs.getVal('flow', false);
-
 				if (flowParam || document.referrer) {
-					//TODO: track middle step for other flows
+					flowTrack.trackFlowStep(flowParam, {editor: editor});
```

Comment:
Maybe we can use approach from last flow step https://github.com/Wikia/app/pull/11546/commits/111e4202ca0964816b0cf67eaf9357dbf3671a37#diff-b8dd1db85320c104bd4af368cc37cea6R4